pkg/skaffold/server/server.go

```diff
@@ -189,29 +191,24 @@ func newGRPCServer(preferredPort int, usedPorts *util.PortSet) (func() error, in
 	}, port, nil
 }
 
-func newHTTPServer(preferredPort, proxyPort int, usedPorts *util.PortSet) (func() error, error) {
```

Comment:
We should

Reply:
probably not. in existing code, the PortSet instance was not shared with the other PortSet instance used in port-forwrding, so it already had no use. the code on the other side already checks for availibility, too, so it seemed redundant to me.